mifosng-android/src/main/java/com/mifos/mifosxdroid/online/CentersActivity.java

```diff
@@ -25,13 +26,20 @@
         .OnFragmentInteractionListener, GroupListFragment.OnFragmentInteractionListener {
 
     private int centerId = 0;
+    private final String LOG_TAG = this.getClass().getSimpleName();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_toolbar_container);
         showBackButton();
-        centerId = getIntent().getExtras().getInt(Constants.CENTER_ID);
+
+        try {
```

Comment:
This is not good idea to put try catch.
Here is Fix 
```java
 @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_toolbar_container);
        showBackButton();
        
        Bundle bundle= getIntent().getExtras();
        if (bundle != null) {
            int centerId = bundle.getInt(Constants.CENTER_ID);
            replaceFragment(GroupListFragment.newInstance(centerId), true, R.id.container);
        } else {
            replaceFragment(new CenterListFragment(), false, R.id.container);
        }
    }
```

and Remove the private int centerId = 0 variable also.